src/main/java/ja/crawler/Entities/BookInfo.java

```diff
@@ -10,4 +14,12 @@ public BookInfo(String title) {
     public String toString() {
         return title;
     }
+
+    public String getTitle() {
```

Comment:
Have you considered Lombok? It would be better looking in the code than just  straight coding getters and setter, you could only add annotations above the field.